ja/lessons/specifics/debugging.md

```diff
@@ -0,0 +1,214 @@
+---
+version: 1.0.1
+title: デバッギング
```

Comment:
I simply translated "Debugging" as "デバッギング", but I thought "デバッグ" might be better because we usually just say "デバッグ" rather than "デバッギング" in Japanese, right? What do you think?